specifications/api/swagger.yaml

```
@@ -1248,6 +1249,17 @@ paths:
         - *parametersWalletId
       responses: *responsesDeleteWallet
 
+  /byron-wallets/{walletId}/transactions:
+    get:
+      operationId: listByronTransactions
+      tags: ["Byron Transactions"]
+      summary: List
+      description: |
+        <p align="right">status: <strong>under testing</strong></p>
+
+        List all incoming and outgoing wallet's transactions.
```

This sentence seems a bit strange. I would write it as this:

```suggestion
        List all incoming and outgoing transactions for the given wallet.
```

---

I would add that the reason it's strange is that it parses as "transactions of incoming wallets and outgoing wallets", when in fact what we want is "incoming transactions and outgoing transactions of a wallet".

---

Done :+1: